full_bubble_model.py

```diff
@@ -747,7 +757,7 @@ def simulate(kwargs):
 
 """________________________________Plotting________________________________"""
 
-def plot(cpar, t_int=np.array([0.0, 1.0]), n=5.0, base_name='', LSODA_timeout=30, Radau_timeout=300, presentation_mode=False, plot_pressure=False, show_legend=False, show_cpar=True):
+def plot(cpar, t_int=np.array([0.0, 1.0]), n=5.0, base_name='', format='png', LSODA_timeout=30, Radau_timeout=300, presentation_mode=False, plot_pressure=False, show_legend=False, show_cpar=True):
```

Comment:
Other formats are now supported. I suggest the use of .svg (vector graphichs allowing arbitrary zooming, and doesn't degrade when converted to pdf)

Reply:
The 2 plots (or 4 if plot_pressure=True) can be saved, if base_name is a non-empty string. 
If format='png' (default), the control parameters are saved as metadata